examples/bigquery_user_info_updater_tool/README.md

```diff
@@ -0,0 +1,317 @@
+# BigQuery User Info Updater
```

Comment:
@jaketf
jaketf on Aug 27, 2019 Member
IMO this is a tool not an example because you could point this asset to any BQ table. what do you think?

 @annarudy
annarudy on Oct 4, 2019 Author Member
that's a good point. Should it go in tools instead?

 @jaketf
jaketf on Oct 17, 2019 Member
yeah please move to tools.

Reply:
moved to tools